### DISCUSSION TITLE
uBlock Run Run Code On Pages not working

### DISCUSSION THREAD

**Discussion by emburnev3** [3kh0/ext-remover#1140] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I put in the commands properly for that exploit, but when I hit ctrl+alt+tilde it didn't work.
My chromebook is version 114.0.5735.358.
![image](https://github.com/user-attachments/assets/6bd1a850-1498-4d37-be1c-fc507a0af0fa)
![image](https://github.com/user-attachments/assets/9ac1c9b3-12c6-4915-ac4a-59adfb6769f9)

**Comment by nathblade16** [comment]
what site are you trying to use it on?

**Reply by emburnev3** [reply]
Was trying on sites like schoology, the new tab(idk what I expected there tbh) a google doc, just places that I would be able to use it on at school
Saw some people using it on like google classrooms but that got disabled(like through the google ACC and can't add another one)

Have been using LTBEEF but that doesn't help bc school wifi blocks stuff and I ant got a phone for mobile hotspot

**Reply by nathblade16** [reply, marked as answer]
try something like example.com

**Reply by S-PScripts** [reply]
no reply yet as of now, marking above as answer until OP says something if ever

**Reply by nathblade16** [reply]
the problem is the csp (content security policy), ublock run uses eval, csps block eval, ublock run cannot be used
